.github/ISSUE_TEMPLATE/releases/release_checklist.md

```diff
@@ -2,151 +2,254 @@
-      - Never overwrite a tarball. At least change the blob storage virtual dir to represent a new build. This can help avoid timing issues and make it more obvious if stale links were accidentally re-sent rather than new ones.
-1.  - [ ] SYNC POINT: Wait for Microsoft build release.
-      - Wait for dotnet/announcements post. <kbd>Watch</kbd> the repo to get emails so you don't have to poll.
+1. - [ ] Upload tarballs and smoke-test prereqs to the `dotnetclimsrc` blob
+         storage under the `source-build` container in `red-hat/3.1/` under a
+         new folder named for the runtime and SDK version.  The access key for
+         this blob storage is in the `EngKeyVault` vault in the "Dotnet
+         Engineering Services" subscription.
+1. - [ ] Complete manual validation steps
+      - For 3.1, the build is poison-checked during the CI build.  If you get
+        an error related to "leak detection" or "CheckForPoison", get the source-build
+        team involved as these can be difficult to diagnose and fix.
+      - [ ] Verify the packs included in the SDK are from source-build-reference-packages,
+        not source-build.
+        - After smoke-tests have run, in `testing-smoke/builtCli`, run
+          `strings sdk/<version>/Sdks/Microsoft.NET.Sdk/tools/net472/Microsoft.CSharp.dll
+          | grep "built by"` and make sure this includes `built by: SOURCEBUILD`.
+      - [ ] Verify the packs included in the SDK have XML documentation comments
+        - From the `testing-smoke/builtCli` directory, run `ls packs/Microsoft.NETCore.App.Ref/<version>/ref/netcoreapp3.1/*.xml`.
+          There should be about 110 XML files here.
+        - From the `testing-smoke/builtCli` directory, run `ls packs/Microsoft.AspNetCore.App.Ref/<version>/ref/netcoreapp3.1/*.xml`.
+          There should be about 130 XML files here.
+        - Issue tracking automation of this: [#1579](https://github.com/dotnet/source-build/issues/1579)
+      - [ ] Re-validate the SHA1s in the `eng/Version.Details.xml` file with
+        the manifest in VSU share dir.
+      - [ ] Review the file list diff between Microsoft-built SDK and source-built
+        SDK in smoke-test output.
+        - For internal builds, smoke-test fails to acquire the Microsoft-built
+          SDK. Manually acquire the Microsoft-built SDK from the build email
+          you should have received, then run `scripts/compare-builds.sh`
+          yourself to get the output to review.
+          - [#1631](https://github.com/dotnet/source-build/issues/1631)
+            tracks a fix.
+        - Use <https://gist.github.com/omajid/9111a9310d452b3594b770d4ca8a3d87>
+          as a loose baseline.
+          - [#1630](https://github.com/dotnet/source-build/issues/1630) tracks
+            a checked-in baseline.
+1. - [ ] Complete manual smoke-testing on our primary testing platforms:
```

Comment:
This should have steps for what is expected here - eg take the tarball, run prep, build, and smoke-tests.

Reply:
Added steps.